src/Storefront/Controller/ComponentsController.php

```diff
@@ -242,13 +243,19 @@ public function removeCard(SalesChannelContext $context, $card) : Response
         ];
 
         $url = Url::getDeleteInstrumentUrl($customerId, $card);
+        
+        try {
+            $deleteCardRequest = Utilities::postRequest( 'DELETE', $url, $header );
 
-        $deleteCardRequest = Utilities::postRequest( 'DELETE', $url, $header );
-
-        return new Response(
-            json_encode(
-                []
-            ), 200, ['Content-Type' => 'text/javascript']
-        );
+            return new Response(
+                json_encode(
+                    []
+                ), 200, ['Content-Type' => 'text/javascript']
+            );
+            
+        } catch (\Exception $e) {
+            $logMessage = Utilities::contructLogBody($e, "cko delete instrument", "checkout.intrument.delete.error", $uuid);
```

Comment:
The question can we recover (this has to be considered for every use-case)?
If not we should throw an exception (and also log it).